app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_create :update_post_comment_counter
+
+  private
+
+  def update_post_comment_counter
+    post.update_column(:comments_counter, post.comments.count)
+  end
```

Comment:
- [ ] [OPTIONAL] Instead of using `update_column` I would suggest you to use `increment!` callback as this callback directly modifies the attribute in the database without reloading the object while `update` method triggers a database update and reloads the object. Example snippet:

```rb
  # app/models/post.rb
  after_save :update_post_counter

  private

 def update_comments_counter
  post.increment!(:comments_counter)
end

```